dark/index.html

```diff
@@ -150,7 +150,7 @@ <h1>
       </div>
       <div id="tooltip">
         <span id="tooltiptext">Telegram</span>
-        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
+        <a href="https://img.icons8.com/fluency/48/telegram-app.png" target="_blank "><img

```

Comment:
You have changed the icon link @maddy020, Don't remove that undo that and change the image source link.
```suggestion
        <a href="https://t.me/YoshithaRathnayake2008" target="_blank "><img
```